packages/@apphosting/adapter-angular/src/utils.ts

```diff
@@ -100,11 +106,12 @@ export function populateOutputBundleOptions(outputPaths: OutputPaths): OutputBun
 export const build = (
   projectRoot = process.cwd(),
   cmd = DEFAULT_COMMAND,
+  ...argv: string[]
 ): Promise<OutputBundleOptions> =>
   new Promise((resolve, reject) => {
     // enable JSON build logs for application builder
     process.env.NG_BUILD_LOGS_JSON = "1";
-    const childProcess = spawn(cmd, ["run", "build"], {
+    const childProcess = spawn(cmd, ["build", ...argv], {
```

Comment:
if cmd = DEFAULT_COMMAND then this line will run ```npm build```. Is that a valid command? I think you might run into this error ```Unknown command: "build"```

Reply:
Ah yeah good catch - I thought `npm build` was still a valid alias for `npm run build` but not anymore. Will adjust accordingly